docs/state.md

```diff
@@ -238,6 +238,83 @@ AppStore
         publisher: string // url of the publisher in question
       }
     },
+    promotion: {
```

Comment:
need to check that if a user modifies/deletes this, they aren't able to re-trigger the promotion and claim funds into the same wallet

Reply:
this is enforced by ledger and grant servers. they both enforce the max of one grant allocated from a given promotionId per wallet.